src/chapters/comm_brevity.tex

```diff
@@ -0,0 +1,260 @@
+\chapter{COMM BREVITY}
+\thumbtab{Comm Brevity}{9}
+\localtableofcontents
+\thispagestyle{plain}
+\cleardoublepage
+
+\section{A-A COMMUNICATION}
+
+\subsection{INTERCEPT}
+
+\begin{tcoloritemize}
+    \blueitem[ANGELS]
+    Height of FRIENDLY aircraft in thousands of feet from MSL.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, AT ANGELS 20.
+        \end{minipage}
+    \end{center}    
+
+    \blueitem[AZIMUTH]
+    1. A picture label describing the lateral separation of two GROUPS.\newline
```

Comment:
no reason for `\newline` here, either just manually add a newline character (blank line in the src), or use `\\`. This will also be solved by swapping to an `enumerate` environment which automatically adds newlines between `\item`s